test-count.ps1

```diff
@@ -0,0 +1,39 @@
+$e2ePath = "Tests/E2E"
+$integrationPath = "Tests/Integration" 
+$unitPath = "Tests/Unit"
+
+$testAttributeRegex = "\[ *(Fact|Theory) *\]"
+
+$outputFile = "test-count.csv"
+$fullOutputPath = Join-Path -Path $PWD -ChildPath $outputFile
+$gitRepoPath = "C:\sources\lora\iotedge-lorawan"
+
+Write-Host "Current directory: " $PWD " OutputPath: " $fullOutputPath
+Clear-Content -Path $fullOutputPath -ErrorAction SilentlyContinue
+Push-Location $gitRepoPath
+Write-Host "Changed to repository directory: " $PWD
```

Comment:
I would write such messages using `Write-Verbose`. It's not usually interesting otherwise.